pkg/notifiers/slack/handle_request.go

```diff
@@ -189,44 +208,59 @@ func (n *Notifier) HandleRequestEvent(ctx context.Context, log *zap.SugaredLogge
 	return nil
 }
 
-func (n *Notifier) UpdateSlackMessage(ctx context.Context, slackClient *slack.Client, log *zap.SugaredLogger, rev access.Reviewer, req access.Request, rule rule.AccessRule, dbRequestor *identity.User) error {
+type UpdateSlackMessageOpts struct {
+	Review            access.Reviewer
+	Request           access.Request
+	RequestReviewerId string
+	Rule              rule.AccessRule
+	DbRequestor       *identity.User
+}
+
+func (n *Notifier) UpdateSlackMessage(ctx context.Context, slackClient *slack.Client, log *zap.SugaredLogger, opts UpdateSlackMessageOpts) error {
 
 	// Skip if requestor == reviewer
-	if rev.ReviewerID == req.RequestedBy {
-		return nil
-	}
+	// if opts.Review.ReviewerID == opts.Request.RequestedBy  {
```

Comment:
@jordiup is this meant to b commented out?
if its not longer required it should be removed

Reply:
Cheers Josh, correct that it can be removed. I've just pushed a commit :)